org.hdfgroup.HDFView.yml

```diff
@@ -85,10 +93,18 @@ modules:
       - -DBUILD_TESTING:BOOL=OFF
       - -DBUILD_SHARED_LIBS:BOOL=ON
       - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk/bin/java
+      - -DJava_JAR_EXECUTABLE=/usr/lib/sdk/openjdk/bin/jar
+      - -DJava_JAVAC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javac
+      - -DJava_JAVAH_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javah
+      - -DJava_JAVADOC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javadoc
+      - -DJAVA_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
+      - -DJAVA_INCLUDE_PATH2=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/linux/
+      - -DJAVA_AWT_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
     sources:
       - type: archive
-        url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.14/hdf5-1.14.3/src/hdf5-1.14.3.tar.bz2
-        sha256: 9425f224ed75d1280bb46d6f26923dd938f9040e7eaebf57e66ec7357c08f917
+        url: https://github.com/HDFGroup/hdf5/releases/download/hdf5_1.14.4.3/hdf5-1.14.4-3.tar.gz
+        sha256: 019ac451d9e1cf89c0482ba2a06f07a46166caf23f60fea5ef3c37724a318e03
```

Comment:
Same here with ``ec2e13c52e60f9a01491bb3158cb3778c985697131fc6a342262d32a26e58e44`` and [hdf5-1.14.5.tar.gz](https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_5/downloads/hdf5-1.14.5.tar.gz)

Reply:
I am sticking with HDF5 1.14.4 because that is what was tested against https://github.com/HDFGroup/hdfview/releases/tag/v3.3.2